splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -74,6 +74,9 @@ else if(v1==v2)
         // Two system procedures are moved, so we need to run base script to update all system procedures
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1928), new UpgradeScriptBase(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1933), new UpgradeScriptToUpdateViewForSYSCONGLOMERATEINSCHEMAS(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1937), new UpgradeScriptForTriggerWhenClause(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,0,0, 1937), new UpgradeScriptForTriggerWhenClause(sdd,tc));
```

Comment:
For master branch, we just need the comparison with 3.1.0.1937, the other two lines are not useful. For branch-3.0 and branch-2.8, we need to compare with 3.0.0.1937 and 2.8.0.1937 respectively, but not all 3.

Reply:
OK, I'm removing the checks at different major/minor releases.